Makefile

```diff
@@ -45,18 +65,38 @@ build: build-scanners build-api
 
 build-api:
 	@echo "### [Building API] ###"
-	@$(CONTAINER_ENGINE) build -t $(API_IMAGE):latest -f src/api/Dockerfile --build-arg="VERSION=${VERSION}" .
+	@$(CONTAINER_ENGINE) build -t $(API_IMAGE):latest -f ./$(DEPLOYMENTS_DIR)/dockerfiles/Dockerfile-api-arg="VERSION=${VERSION}" .
 	@$(CONTAINER_ENGINE) tag $(API_IMAGE):latest $(API_IMAGE):$(VERSION)
 	@$(CONTAINER_ENGINE) tag $(API_IMAGE):latest $(API_IMAGE):$(IMAGE_TAG)
 
 build-scanners: build-aws-scanner
 
 build-aws-scanner:
 	@echo "### [Building AWS Scanner] ###"
-	@$(CONTAINER_ENGINE) build -t $(SCANNER_IMAGE):latest -f src/aws_scanner/Dockerfile --build-arg="VERSION=${VERSION}" .
+	@$(CONTAINER_ENGINE) build -t $(SCANNER_IMAGE):latest -f ./$(DEPLOYMENTS_DIR)/dockerfiles/Dockerfile-scanner-aws --build-arg="VERSION=${VERSION}" .
 	@$(CONTAINER_ENGINE) tag $(SCANNER_IMAGE):latest $(SCANNER_IMAGE):$(VERSION)
 	@$(CONTAINER_ENGINE) tag $(SCANNER_IMAGE):latest $(SCANNER_IMAGE):$(IMAGE_TAG)
 
+
+# Building in local environment
+local-clean:
+	@echo "### [Cleanning local building] ###"
+	@rm -Rf $(BIN_DIR)
```

Comment:
Maybe `BUILD_DIR` instead of?

Reply:
I thought so, but read that /build dir also could contain packaging or CI/CD files, and I wanted to prevent to remove those files in case we include them on the future.